src/main/java/org/jetbrains/research/refactorinsight/folding/handlers/MoveOperationHandler.java

```diff
@@ -0,0 +1,45 @@
+package org.jetbrains.research.refactorinsight.folding.handlers;
+
+import com.intellij.psi.PsiFile;
+import com.intellij.psi.PsiMethod;
+import org.jetbrains.research.refactorinsight.data.RefactoringInfo;
+import org.jetbrains.research.refactorinsight.folding.JavaFileFinder;
+
+import java.util.Collections;
+import java.util.List;
+
+public class MoveOperationHandler implements FoldingHandler {
```

Comment:
We already have a class with the same name, it might be a bit confusing

Reply:
May be renamed to MoveOperationFoldingHandler or rename FoldingHandler to DiffViewHandler because in future some refactorings may only add hint, not folding